staff/sergio-costa/cv/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Sergio Costa</title>
+</head>
+<body>
+    <header>
+        <h1>CV Sergio Costa</h1>
+    </header>
+    <section>
+        <h3>Datos personales</h3>
+        <ul>
+            <li><b>Direccion:</b> C/Josep Irla 27</li>
```

Comment:
The element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the to
tags, emphasized text should be denoted with the tag, important text should be denoted with the tag, and marked/highlighted text should use the tag. https://www.w3schools.com/tags/tag_b.asp

Reply:
fixed 👍